README.rst

```diff
@@ -26,13 +27,13 @@ Run integration tests against stage deployment:
 
 .. code-block:: console
 
-  THOTH_USER_API_HOST=stage.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.stage.thoth-station.ninja ./test.sh
+  THOTH_USER_API_HOST=stage.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.stage.thoth-station.ninja THOTH_AMUN_API_HOST=amun.stage.thoth-station.ninja ./test.sh
```

Comment:
Do we have such endpoint like `amun.stage.thoth-station.ninja`?

Reply:
we do , its here: https://console-openshift-console.apps.ocp.prod.psi.redhat.com/k8s/ns/thoth-amun-api-stage/routes
however we need to direct route53 to ocp cluster